napari/layers/labels/labels.py

```diff
@@ -937,10 +943,77 @@ def _partial_labels_refresh(self):
         colors_sliced = self._raw_to_displayed(
             raw_displayed, data_slice=updated_slice
         )
+        self._slice.image.view[updated_slice] = colors_sliced
```

Comment:
I do not know why I need to this line. On main the `view` array is updated, but I cannot find where. But After I add this the test are passing.

Reply:
@Czaki 👆?